Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -244,7 +264,28 @@ public void AddRow( DataGridRowInfo<TItem> row )
     /// </summary>
     /// <param name="column">Column to link with this datagrid.</param>
     public bool RemoveColumn( DataGridColumn<TItem> column )
-        => Columns.Remove( column );
+    {
+        var removed = Columns.Remove( column );
+        
+        if ( column.SortDirection != SortDirection.Default )
+        {
+            SortByColumns.Remove( column );
+
+            _ = InvokeAsync( () =>
+                SortOrderChanged.InvokeAsync(
+                    new DataGridSortOrderChangedEventArgs(
+                        SortMode,
+                        SortByColumns.Select( c => new DataGridSortInfo(
+                                c.Field,
+                                c.GetFieldToSort(),
+                                c.CurrentSortDirection,
+                                c.SortOrder ) )
+                            .OrderBy( c => c.SortOrder )
+                            .ToList() ) ) );

```

Comment:
Do we think this is expected? If a column is removed should it realy trigger a Sort Order Changed?